docs/api-testing/vault.md

```diff
@@ -129,21 +129,11 @@ For each snippet, two actions are available:
 A good use case for the snippets feature is building an authentication flow; you don't need nor want to rewrite all authentication steps for every single test. Instead, call the snippet that contains these authentication details. Another good example is integration testing, where you can reuse various tests to create one larger flow.
 
 Below is an example of how to create an Authentication Snippet.
-
-1. First, create a new test with a request component that requires basic authentication. For examples, check the [Sauce Labs REST API endpoints](/dev/api/) for ideas.
-   <img src={useBaseUrl('img/api-fortress/2021/04/exampleSnippetRequest.png')} alt="Example Snippet Request"/>
-
-1. Select to the **+ Add Request Headers** section below the request component
-   <img src={useBaseUrl('img/api-fortress/2021/04/addRequestHeader.png')} alt="Add Request Header"/>
-
-1. Select **Basic Authentication** from the list
-   <img src={useBaseUrl('img/api-fortress/2021/04/basicAuth.png')} alt="Basic Auth Component"/>
-
-1. Enter the details for `username` and `password`, then select **Save**.
-   <img src={useBaseUrl('img/api-fortress/2021/04/basicAuthDetails.png')} alt="Basic Auth Details Component"/>
-
-1. Once the **Authorization Header** appears, highlight it in the UI, then select the **Export snippet from selection** icon in the toolbar.
-   <img src={useBaseUrl('img/api-fortress/2021/04/authSnippet.png')} alt="Auth Snippet screenshot"/>
+1. First, create a new test with a request component that requires basic authentication. For examples, check the [Sauce Labs REST API endpoints](/dev/api/) for ideas.<br/><img src={useBaseUrl('img/api-fortress/2021/04/exampleSnippetRequest.png')} alt="Example Snippet Request"/>
```

Comment:
@kimsauce it's not an ordered list 😬

Reply:
@LoryApiFortress In markdown, you can use 1s, and it renders as an ordered list -- this is the output

<img width="657" alt="Screen Shot 2021-11-19 at 8 41 02 AM" src="https://user-images.githubusercontent.com/56411016/142659654-42f1d6b8-305d-4665-b745-a5746f1f7836.png">
:

Reply:
@kimsauce ops I didn't know this 😄  I looked only to the code